routers/employee.py

```diff
@@ -348,8 +348,6 @@ async def user_exists(request: Request, employee_id: str, db: Session = Depends(
     if user is None:
         return RedirectResponse(url="/admin/login", status_code=status.HTTP_302_FOUND)
 
-    role_state = db.query(models.Roles).filter(models.Roles.id == user['role_id']).first()
```

Comment:
Role state needs to remain as a future update when a 'user already exists' state is triggered both employee details will be displayed for comparison so the options to update/re-onboard will become available.

role_state is also needed for display navbar options.

Reply:
I see now this was a duplicate entry. Thank you for spotting it.